rootfs/network/parse-config

```diff
@@ -103,9 +103,18 @@ EOF
     else
         echo 'iface eth0 inet dhcp'
         echo -n '    udhcpc_opts -t 10 -O search'
+        if HOSTNAME="$(parse_config HOSTNAME)"; then
+            # Send hostname to DHCP server
+            echo -n " -x hostname:$HOSTNAME"
+        else
+            # Ask for hostname from DHCP server
+            echo -n " -O hostname"
+        fi
         if NTP="$(parse_config NTP)"; then
+            # Use defined NTP servers
```

Comment:
We now have a chain of `echo -n` calls followed by `echo`, but in two different ways.  Now would be a good time to rewrite
```sh
        if NTP="$(parse_config NTP)"; then
            echo
        else
            echo ' -O timesrv'
        fi
```
as
```sh
        if ! NTP="$(parse_config NTP)"; then
            echo -n ' -O timesrv'
        fi
        echo
```
This makes the need for the final `echo` clearer.

(Do double check the `if !` syntax works on PandA, but I'm pretty sure it does.)